starlette_admin/templates/forms/simplemde.html

```diff
@@ -0,0 +1,10 @@
+<div class="{% if error %}{{ field.error_class }}{% endif %}">
+    <textarea name="{{ field.id }}" data-bs-toggle="autosize" class="{{ field.class_ }}
+            {% if error %}{{ field.error_class }}{% endif %}" {{ field.input_params() | safe }}
+            config="{{ field.config() }}"
```

Comment:
I think we can merge `config` with `field.input_params()` by overriding `input_params()` and adding the `config` attribute. A new form template is not neccessary

Reply:
This field is a subclass of TextAreaField and inherits these parameters (attributes to be passed to the textarea tag): `rows`, `minlength`, `maxlength`, `placeholder`, `required`, `disabled`, `readonly`. These values are passed through `input_params` method.

Only `placeholder` attribute still works here. Since the `input_params` method is used to pass `tag attributes` in many places, I believe we shouldn't use it for another purpose.

People might want to use it as it is, as expected, without searching about any configuration about the SimpleMDE in this case.

One last thing, I believe using `input_params` for this will be a little bit "confusing" in the future. Someone may try to pass tag attribute using this but you're recommendation makes it different. I can rename it into `options` if you want.

I think we shouldn't limit the users here but if you insist I can change it.

Reply:
The purpose of the `input_params` is to make it easier to add extra  HTML  attributes to the input, without having to overwrite the form template. This is why many fields share the `input.html` template. 
Here is my suggestion:
```python
class SimpleMDEField(TextAreaField):
    def input_params(self) -> str:
        return super().input_params() + " " + html_params({"data-config": json.dumps(...)})
```

Reply:
> The purpose of the `input_params` is to make it easier to add extra HTML attributes to the input, without having to overwrite the form template.

I'm sorry for my bad English, I was trying to say the same thing lol.




> This is why many fields share the `input.html` template. Here is my suggestion:
> 
> ```python
> class SimpleMDEField(TextAreaField):
>     def input_params(self) -> str:
>         return super().input_params() + " " + html_params({"data-config": json.dumps(...)})
> ```

I just commit some code, can you check them? I did it before seeing your recommendation.

Reply:
👋 @jowilf I agree with you now. I have made the changes that you requested.